feature/home/src/main/java/org/sopt/official/feature/home/component/HomeEnjoySoptServicesBlock.kt

```diff
@@ -0,0 +1,179 @@
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.Center
+import androidx.compose.ui.Alignment.Companion.TopEnd
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
+import org.sopt.official.designsystem.Orange400
+import org.sopt.official.designsystem.SoptTheme
+import org.sopt.official.designsystem.SoptTheme.colors
+import org.sopt.official.designsystem.SoptTheme.typography
+import org.sopt.official.designsystem.component.UrlImage
+import org.sopt.official.feature.home.model.HomeAppService
+
+@Composable
+internal fun HomeEnjoySoptServicesBlock(
+    appServices: ImmutableList<HomeAppService>,
+    onAppServiceClick: (url: String) -> Unit,
+) {
+    Text(
+        text = "SOPT 더 재밌게 즐기기!",
+        style = typography.heading20B,
+        color = colors.onBackground,
+    )
+    Spacer(modifier = Modifier.height(height = 16.dp))
+    LazyRow(
+        horizontalArrangement = Arrangement.spacedBy(space = 16.dp),
+        modifier = Modifier.fillMaxWidth(),
+    ) {
+        items(appServices) { appService ->
+            AppServiceItem(
+                appService = appService,
+                onItemClick = onAppServiceClick,
+            )
+        }
+    }
+}
+
+@Preview
+@Composable
+private fun HomeEnjoySoptServicesBlockPreview() {
+    SoptTheme {
+        Column {
+            HomeEnjoySoptServicesBlock(
+                appServices = PREVIEW_FIXTURE,
+                onAppServiceClick = {},
+            )
+        }
+    }
+}
+
+@Composable
+private fun AppServiceItem(
+    appService: HomeAppService,
+    onItemClick: (url: String) -> Unit,
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.clickable { onItemClick(appService.deepLink) }
+    ) {
+        Box(contentAlignment = TopEnd) {
+            HomeButtonCircleBox {
+                UrlImage(
+                    url = appService.iconUrl,
+                    modifier = Modifier.size(size = 60.dp),
+                )
+            }
+            if (appService.isShowAlarmBadge) AppServiceAlarmBadge(text = appService.alarmBadgeContent)
```

Comment:
```suggestion
            if (appService.isShowAlarmBadge) {
                AppServiceAlarmBadge(text = appService.alarmBadgeContent)
            }
```

이건 개인차긴 한데 코드 블럭들은 중괄호로 표현하는게 일관성 있지 않을까 싶어. 어떻게 생각하시오?

Reply:
간결성 vs 일관성